src/pages/task/actionButtons/previousCommits/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { getTaskRoute } from "constants/routes";
+import { LastMainlineCommitQuery } from "gql/generated/types";
+import { reportError } from "utils/errorReporting";
+import { CommitTask, CommitType } from "./types";
+
+// a link cannot be null, so it's common to use # as a substitute.
+const nullLink = "#";
+
+export const getLinks = ({
+  lastExecutedTask,
+  lastPassingTask,
+  parentTask,
+}: {
+  lastExecutedTask: CommitTask;
+  lastPassingTask: CommitTask;
+  parentTask: CommitTask;
+}) => {
+  if (!parentTask) {
+    return {
+      [CommitType.Base]: nullLink,
+      [CommitType.LastPassing]: nullLink,
+      [CommitType.LastExecuted]: nullLink,
+    };
+  }
+
+  return {
+    [CommitType.Base]: getTaskRoute(parentTask.id),
+    [CommitType.LastPassing]: getTaskRoute(
```

Comment:
This link would always default to the parent task even if it does not pass

Reply:
Sorry if I'm misunderstanding, doesn't [this test case](https://github.com/evergreen-ci/spruce/blob/dee29eda9ca94e917eb2407514e818bb630fcd46/src/pages/task/actionButtons/previousCommits/PreviousCommits.test.tsx#L120) assert that the last passing task works correctly?

Reply:
Thanks for looping back I missed that!